<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeMovieDialog</class>
 <widget class="QWidget" name="MakeMovieDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="verticalWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>120</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Frame width:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_width">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Original</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Scale</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1080</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>720</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>640</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>480</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>320</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>240</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Frame height:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_height">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Original</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Scale</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>720</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>640</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>480</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>320</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>240</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Fps:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_fps">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>60</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>40</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>25</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>15</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Transform:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_transform">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reverse</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mirror</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Loop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Insert keyframes</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_create">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
